Automattic/themes
# Add PHPCS and PHPCBF to fix linting issues

This adds a PHPCS check before you commit a file. It will also try to automatically fix any issues it finds in the file for you.

It doesn't block you from committing if there are errors. It also doesn't work unless you run `composer install` in your root directory. I hope this means that contributors won't be preventing from committing, while enabling us to improve code quality.

To test:
- run `npm i` in the root of the repo
- then run `composer install`
- make a change to a file
- commit the file
- notice that any problems with the coding standards in the file are reported to you.
